packages/cli/src/util/deploy/create-git-meta.ts

```diff
@@ -0,0 +1,155 @@
+import fs from 'fs-extra';
+import { join } from 'path';
+import ini from 'ini';
+import git from 'git-last-commit';
+import {
+  BitbucketMeta,
+  GitHubMeta,
+  GitLabMeta,
+  GitMeta,
+  RepoData,
+} from '../../types';
+import { Output } from '../output';
+
+function getLastCommit(directory: string): Promise<git.Commit> {
+  return new Promise((resolve, reject) => {
+    git.getLastCommit(
+      (err, commit) => {
+        if (err) return reject(err);
+        resolve(commit);
+      },
+      { dst: directory }
+    );
+  });
+}
+
+export async function getRepoData(configPath: string, output: Output) {
+  let gitConfig;
+  try {
+    gitConfig = ini.parse(await fs.readFile(configPath, 'utf-8'));
+  } catch (error) {
+    output.debug(`Error while parsing repo data: ${error.message}`);
+  }
+  if (!gitConfig) {
+    return;
+  }
+
+  const originUrl = gitConfig['remote "origin"']?.url;
+  if (originUrl) {
+    return parseRepoUrl(originUrl);
+  }
+}
+
+export function parseRepoUrl(originUrl: string): RepoData | null {
+  const isSSH = originUrl.startsWith('git@');
+  // Matches all characters between (// or @) and (.com or .org)
+  const provider = originUrl.match(/(?<=(\/\/|@)).*(?=(.com|.org))/);
```

Comment:
```suggestion
  const provider = originUrl.match(/(?<=(\/\/|@)).*(?=(\.com|\.org))/);
```

Pretty sure these dots need to be escaped.

Lets add a unit test for `https://examplecom/foo` which would match right now but it shouldn't match

Reply:
Thank you for catching this!